src/canmatrix/canmatrix.py

```diff
@@ -164,16 +164,16 @@ def setDefaultMax(self):
         return  self.setMax()
 
     unit = attr.ib(type=str, default ="")
-    receiver = attr.ib(default = attr.Factory(list))
+    receiver = attr.ib(type=list, factory=list)
```

Comment:
The linked file says about typing.list: "Useful for annotating return types. To annotate arguments it is preferred to use abstract collection types such as Mapping, Sequence, or AbstractSet." 
This is not "an iterable arument" but a variable of `list` type. It's rather 'return type' like return from a `@property` :-)

Reply:
I don't know the inner workings of typing with attrs but this certainly results in an `__init__()` which accepts a parameter.  You also get an instance attribute which, if nothing was passed, is an empty list.  My overall point is that there still isn't agreement on exactly how this should be handled so tossing it in with other changes will just hold up the other changes.

Reply:
Since we do `.append()` to `self.receiver` then I suppose [`typing.MutableSequence`](https://docs.python.org/3/library/typing.html#typing.MutableSequence) is probably more appropriate than just sequence.  Though it seems more like it should be a `set` with how it's managed.

Reply:
You're right as always, my point of view was wrong. I still doubt that someone would pass something else than a list in the `__init__` for this particular attribute but I agree, that in general the list type could be too restricting.